pkgs/by-name/ti/tinyxxd/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  installShellFiles,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "tinyxxd";
+  version = "1.3.5";
+
+  src = fetchFromGitHub {
+    repo = "tinyxxd";
+    owner = "xyproto";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-W7BrQga98ACrhTHF3UlGQMRmcdJaxgorDP6FpD5mr2A=";
+  };
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  installFlags = [ "PREFIX=$(out)" ];
+
+  postInstall = ''
+    installManPage tinyxxd.1
+
+    # Allow using `tinyxxd` as `xxd`. This is similar to the Arch packaging.
+    # https://gitlab.archlinux.org/archlinux/packaging/packages/tinyxxd/-/blob/main/PKGBUILD
+    ln -s $out/bin/{tiny,}xxd
+    ln -s $out/share/man/man1/{tiny,}xxd.1.gz
+  '';
+
+  meta = {
+    homepage = "https://github.com/xyproto/tinyxxd";
+    description = "Drop-in replacement and standalone version of the hex dump utility that comes with ViM";
+    license = [
+      lib.licenses.mit
+      lib.licenses.gpl2Only
+    ];
```

Comment:
This implies `MIT AND GPL-2.0-only`, whereas it’s actually `OR`. The two options used in‐tree here are `license = lib.licenses.mit; # OR GPL-2.0-only` (my preference, as we can choose which licence to use as a downstream, and the GPL’s requirements are a superset of MIT’s), or `license = [ lib.licenses.mit /*or*/ lib.licenses.gpl2Only ];` (somewhat incoherent IMO, if you consider how it works with a case where a package is available under e.g. either the AGPL or an unfree licence, but more popular in the tree despite this).

We really need to just encode full SPDX expressions already…

Reply:
I'll do that edit.